bitbucket/views.py

```diff
@@ -13,8 +13,16 @@ class BitbucketReposView(APIView):
 
     def get(self, request, project_id):
         data = request.data
-        data['project_id'] = project_id
-        data.update({'katka_project_id': request.query_params.get('katka_project_id')})
+        if request.query_params.get('limit') is not None:
```

Comment:
minor: in the service there is already a check if the `limit` and `start` are None. You could skip the direct ifs here
```
data['limit'] = request.query_params.get('limit', None)
data['start'] = request.query_params.get('start', None)
```

Reply:
I am doing this since None is not a valid value for integer serializer field.

Reply:
ah, ok then!